examples/textures/05-image-drawing.lisp

```diff
@@ -67,14 +64,16 @@
                         (image-crop cat-image cat-crop)
                         (image-flip-horizontal cat-image)
                         (image-resize cat-image 150 200)
-                        (draw-image-on-image cat-image parrots-image)
+                        ;; FIXME neither this ↑ or this ↓ resize seems to work, confusingly
+                        ;; (claylib/ll:image-resize (claylib::c-struct cat-image) 150 200)
+                        (image-draw parrots-image cat-image)
```

Comment:
Here is the thing I am hung up on at the moment: before this commit I had an `image-resize` pass-through created with `defun-pt`, but now I have written it manually to accept either an `rl-image` or the new claylib `image` class. Same goes for other image manipulation pass-throughs. The odd thing is neither the new `image-resize` nor using `claylib/ll:image-resize` directly appears to be working.

Reply:
Hrmmm. What's the definition of "not working"?

Reply:
It runs without crashing, but simply fails to resize the cat image 
![image](https://user-images.githubusercontent.com/34904034/173193902-3e1fb3aa-334e-4ac8-870c-e31035609bd0.png)

Reply:
This sounds like one of those annoying bugs that's actually in a different place in the code than you think it should be. I looked at your `image-resize` and didn't see anything obviously wrong with it.

Have you tried switching back to the `defun-pt` version, deleting FASL's, and recompiling? I'm going to hazard a guess that it's still not going to work and you'll be stuck digging through the rest of your changes. Would love to be wrong.

Reply:
> Have you tried switching back to the defun-pt version, deleting FASL's, and recompiling?

Yes I have tried that. I think you're right that the issue is somewhere unexpected. I'll try to figure it out.

Reply:
Silly mistake. Fixed!

Reply:
Saw the change, looks like you were trying to reference asset fields and it broke because of the futures stuff?